static/panes/editor.ts

```diff
@@ -1912,10 +1912,10 @@ export class Editor extends MonacoPane<monaco.editor.IStandaloneCodeEditor, Edit
     }
 
     override getPaneName(): string {
-        if (this.filename) {
-            return this.filename;
-        } else if (this.paneName) {
+        if (this.paneName) {
             return this.paneName;
+        } else if (this.filename) {
```

Comment:
We really need `getPaneName` to actually return the paneName, in particular when both paneName and filename are present.